product/views.py

```diff
@@ -1,3 +1,128 @@
-from django.shortcuts import render
+import json
 
-# Create your views here.
+from django.views     import View
+from django.http      import JsonResponse
+from django.db.models import Q, Min
+
+from .models          import Product, Image, Size, ProductSize 
+from order.models     import Ask, Bid, OrderStatus, ExpirationType
+
+ORDER_STATUS_CHECK   = 'current'
+ORDER_STATUS_HISTORY = 'history'
+
+class ProductListView(View):
+    def get(self, request):
+        sizes         = Size.objects.all()
+        lowest_price  = request.GET.get('lowest')
+        highest_price = request.GET.get('highest')
+        limit         = int(request.GET.get('limit', 0))
+        offset        = int(request.GET.get('offset', 0))
+        
+        products = Product.objects.all()
+
+        total_products = [
+            {'productId'   : product.id,
+            'productName'  : product.name,
+            'productImage' : product.image_set.all()[0].image_url,
```

Comment:
```suggestion
            'productImage' : product.image_set.first().image_url,
```
모든 객체 가운데 첫번째 요소만 선택하길 원하신다면 `first()` 메소드를 추천드립니다 :)

Reply:
네 변경했습니다~